### ISSUE TITLE
Map search role removal

### ISSUE THREAD

**Issue by sarah-beasley** [open-sdg/open-sdg#1849]
When using the screenreader on the map search box, it currently has a 'role=search' code. Have been informed by DAC that this is incorrect and breaking the screen reader process apparently so needs to be removed.

![image](https://user-images.githubusercontent.com/106670682/198000621-85abb6ee-a669-4041-9068-bc7b5d6acd44.png)

**Comment by brockfanning** [comment]
@sarah-beasley This has come up in the past, and if I remember right, we were never able to replicate it. When I go to look at the page, and I inspect that element, it does not have `role="search"`. From the screenshot I clearly see it though - I just don't know how that is showing up. Maybe it is a particular browser, or maybe some accessibility tool is adding that. But at the moment I'm stumped here.

**Comment by sarah-beasley** [comment]
@brockfanning hmm that's interesting... I just tried on microsoft edge on the live site and on beta3 and it is still coming up for me when i right click in the map search bar... so that's google chrome and edge that have it. are you using a different browser?

**Comment by brockfanning** [comment]
@sarah-beasley Aha! Yes I use Firefox - the problem doesn't happen in Firefox for some reason. Well that solves the mystery. :) I just put up a PR for that.
